### PR TITLE
temp change repo

### PR DESCRIPTION
**Pull request by enoch85**
change it back once in nextcloud master

### CODE REVIEW

**Comment by enoch85** [comment]
cc @morph027 please merge

**Comment by enoch85** [comment]
ping @morph027 need this to test properly.

File: nextcloud-startup-script.sh [hunk 1]
```diff
@@ -658,8 +654,6 @@ apt autoremove -y
 apt autoclean
 CLEARBOOT=$(dpkg -l linux-* | awk '/^ii/{ print $2}' | grep -v -e "$(uname -r | cut -f1,2 -d"-")" | grep -e "[0-9]" | xargs sudo apt -y purge)
 echo "$CLEARBOOT"
-find /root "/home/$UNIXUSER" -type f \( -name '*.sh*' -o -name '*.html*' -o name '*.tar*' -o name '*.zip*' \) -delete
-clear
```
Comment (enoch85) [inline nextcloud-startup-script.sh]: this exists in the first cleanup

**Comment by enoch85** [comment]
if I run  sudo bash nextcloud-startup.script.sh` when the installation script is done, it says that I'm not root. So with your check, sudo doesn't get accepted as root.

Also, changed the function of contacts webmin and calendar so that they are seperate scripts. But wget each file in the `while` command doesn't work. Do you have a better soulution @morph027

**Comment by enoch85** [comment]
`install_3rdparty_app` works but not `install_app`

**Comment by enoch85** [comment]
OK, just realized this needs to be in @morph027 master to work properly, can't test out of my branch as the lib.sh will not contain the needed $vars *duuh*.

@morph027 Can you please merge this so we can fix the errors later? Thanks!

**Comment by enoch85** [comment]
**ERRORS FOUND DURING TESTING**

- [ ] Every time lib.shis loaded this shows up:
```
cat: /var/mysql_password.txt: No such file or directory
sudo: php: command not found
```
- [ ] chown command is wrong in ncadmin
- [ ] sudo bash doesn't work when the installed script is run, and it doesn't recognize that I'm root

**Comment by morph027** [comment]
Fixed:

> [x] chown command is wrong in ncadmin

**Comment by morph027** [comment]
> sudo bash doesn't work when the installed script is run, and it doesn't recognize that I'm root

Hm, weird. If i'm running my test script, it works...

```bash
cat /tmp/foo.sh
#!/bin/bash

is_root() {
    if [[ "$EUID" -ne 0 ]]
    then
        return 1
    else
        return 0
    fi
}

is_root && echo root || echo noroot
```

```bash
$ /tmp/foo.sh
noroot
$ sudo /tmp/foo.sh
root
```

**Comment by morph027** [comment]
> Every time lib.shis loaded this shows up:
> ```
> cat: /var/mysql_password.txt: No such file or directory
> sudo: php: command not found
> ```

Also fixed in master.

**Comment by enoch85** [comment]
Great! Thanks!

Can you have a look at this as well? 

``` 
/dev/fd/63: line 36: read: `Do you want to create a new user? ([y]es or [N]o): ': not a valid identifier
```
It''s everywhere so I think it is in the variable itself.

**Comment by enoch85** [comment]
Also, this:

```
System config value preview_libreoffice_path set to string /usr/bin/libreoffice
+ whiptail --title 'Which apps/programs do you want to install?' --checklist --separate-output '' 10 40 3 Calendar '              ' on Contacts '              ' on Webmin '              ' on
+ read -r -u 9 choice
+ case "$choice" in
+ wget -q https://raw.githubusercontent.com/morph027/vm/master/static/calendar.sh -P /var/scripts
+ install_app calendar
nextcloud_install_production.sh: line 418: install_app: command not found
```

**Comment by enoch85** [comment]
Same for contacts I think

**Comment by morph027** [comment]
> ```/dev/fd/63: line 36: read: `Do you want to create a new user? ([y]es or [N]o): ': not a valid identifier```

Fixed...

**Comment by morph027** [comment]
> ```nextcloud_install_production.sh: line 418: install_app: command not found```

Fixed...

**Pull request merged**